docs/usage/shoot_updates.md

```diff
@@ -0,0 +1,93 @@
+# Shoot Updates and Upgrades
+
+This document describes what happens during shoot updates (changes incorporated in a newly deployed Gardener version) and during shoot upgrades (changes for version controllable by end-users).
+
+## Updates
+
+Updates to all aspects of the shoot cluster happen when the gardenlet reconciles the `Shoot` resource.
+
+### When are Reconciliations Triggered
+
+Generally, when you change the specification of your `Shoot` the reconciliation will start immediately, potentially updating your cluster.
+
+You can also annotate your shoot with special operation annotations (see [this document](shoot_operations.md)) which will cause the reconciliation to start due to your actions.
+
+There is also an automatic reconciliation by Gardener.
+The period, i.e., how often it is performed, depends on the configuration of the Gardener administrators/operators.
+In some Gardener installations the operators might enable "reconciliation in maintenance time window only" ([more information](shoot_maintenance.md#cluster-reconciliation)) which will result in at least one reconciliation during the time configured in the `Shoot`'s `.spec.maintenance.timeWindow` field.
+
+Please note that you can also confine the reconciliation triggered due to your specification updates to the cluster's maintenance time window. Please find more information [here](shoot_maintenance.md#confine-specification-changesupdates-roll-out).
+
+### Which Updates are Applied
+
+As end-users can only control the `Shoot` resource's specification but not the used Gardener version, they don't have any influence on which of the updates are rolled out (other than those settings configurable in the `Shoot`).
+A Gardener operator can deploy a new Gardener version at any point in time.
+Any subsequent reconciliation of `Shoot`s will update them by rolling out the changes incorporated in this new Gardener version.
+
+Some examples for such shoot updates are:
+
+* Add a new/remove an old component to/from the shoot's control plane running in the seed, or to/from the shoot's system components running on the worker nodes.
+* Change the configuration of an existing control plane/system component.
+* Add a new/remove an old condition to/from the `.status.conditions[]` list maintained in the `Shoot` resource.
```

Comment:
This might a bit confusing FMPOV, as new Shoot conditions will not be set initially by a reconciliation, but rather as soon as the care controller acts on a Shoot, which might be way before a reconciliation.

Reply:
Makes sense, I remove it.